recipe/meta.yaml

```diff
@@ -24,20 +25,22 @@ requirements:
     - setuptools
     - blas 1.1 {{ variant }}
     - openblas 0.2.20|0.2.20.*
-    - gsl     # [not win]
-    - fftw    # [not win]
+    - gsl
+    - fftw
+    - dsdp   # [not win]
     - glpk
-    - suitesparse 4.5.6   # [not win]
+    - suitesparse 4.5.6|4.5.6.*   # [not win]
     - suitesparse 5.1.*   # [win]
```

Comment:
Could you please raise a [`conda-forge-pinning` issue]( https://github.com/conda-forge/conda-forge-pinning-feedstock/issues ) about pinning `suitesparse` globally?

Reply:
Basically `conda-forge-pinning` determines how things get pinned globally so that we don't need to hardcode them ourselves. So it would be good if it incorporated a sensible pinning for `suitesparse`. Raising the issue there would put adding this to the [pinning list]( https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml ) on other people's radar.

ref: https://conda-forge.org/docs/meta.html#pinning-packages

Reply:
Also please take a look at the `.ci_support` files that the re-rendering bot added. This will help give an idea as to how dependencies are handled now.

Reply:
See conda-forge/conda-forge-pinning-feedstock#59